tasks/bootstrap_local_vm/05_add_host.yml

```diff
@@ -126,16 +126,47 @@
     retries: 120
     delay: 5
   - debug: var=host_result_up_check
-  - name: Check host status
-    fail:
-      msg: >
-        The host has been set in non_operational status,
-        please check engine logs,
-        fix accordingly and re-deploy.
+  - name: Handle non_operational failure
```

Comment:
Rephrase to:
Handle deployment failure

Try to be less technical in the name descriptions and messages to the user.
Try to avoid using variable names in those places too.

Reply:
done